docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc

```diff
@@ -0,0 +1,50 @@
+= Component deployment lifecycle
+CP Development Team <cp-devel@redhat.com>
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Scope
+Learn how to track the lifecycle of a component's deployment in the {ProductName} web console.
+
+== Web console visualization
```

Comment:
I might move this section to the end of the doc. I think both 'build pipeline customization' and 'logs' might get lost at the bottom underneath the table. 
On the other hand, this makes the most sense as the first article in this guide.

Reply:
I'm moving the table to the end of the doc :)